### ISSUE TITLE
Sometimes dir creation is raising a PHP warning

### ISSUE THREAD

**Issue by Velliz** [Velliz/pukoconsole#1]
on release build ```0.1.0``` when executing command:

```php puko routes service add api/tests```

i got:

```
PHP Warning:  mkdir(): File exists in D:\xampp\htdocs\bbnv2\vendor\puko\console\src\Routes.php on line 193
Warning: mkdir(): File exists in D:\xampp\htdocs\bbnv2\vendor\puko\console\src\Routes.php on line 193
```

but the file generated successfully.

**Comment by Velliz** [comment]
Warning: mkdir(): File exists in \puko\console\src\Routes.php on line 193
Todo: 
provide a separate global function for dir creation.
*happening for routes generation that .php controller file don't exists only.

**Comment by Velliz** [comment]
Already fixed in commits ae6ada0
